src/components/m-table-edit-field.js

```diff
@@ -87,6 +89,7 @@ class MTableEditField extends React.Component {
       <MuiPickersUtilsProvider utils={DateFnsUtils}>
         <DateTimePicker
           {...this.getProps()}
+          {...this.props.localization}
```

Comment:
Thanks,
Can you `{...this.props.localization}` move to `this.getProps()` inside